package.json

```diff
@@ -259,6 +259,14 @@
             }
         },
         "commands": [
+            {
+                "command": "vscode-neovim.toggle",
+                "title": "Neovim Toggle: Stay in InsertMode Forever"
+            },
+            {
+                "command": "vscode-neovim.toggle2",
+                "title": "Neovim Toggle: Stay in InsertMode OnBufferChange"
```

Comment:
could take an arg instead of adding an extra command?

Reply:
Conceptually, I think it would be better if the second toggle (stay in InsertMode on BufferChange) was a setting, since it's more of a decision you make once and don't change that often. The first toggle, on the other hand, should be a toggle. If you agree, I will add the second toggle to the settings.
But I think I need your help for that, because when I tried to add a setting some time ago, it seemed like I implemented it wrong or something.

Reply:
`toggle2` will probably run into https://github.com/vscode-neovim/vscode-neovim/issues/642. I suggest to focus on implementing just `toggle` (forever) to avoid [scope creep](https://mtlynch.io/code-review-love/#5-narrowly-scope-changes).

Reply:
I think you are right. I will remove all the StayInInsertModeOnBufferChange functionality from this pull request. But I'd still like to implement it in another pull request. In the end, after implementing both functions, I won't touch any of it because StayInInsertModeOnBufferChange is what I really want, and once set it's super handy. This way, the transition from normal Vscode to using the keyboard over the extension feels very smooth to me. When colleagues are remotely connected or I need to do something in normal Vscode fashion, I just press "i"... and as long as I don't press Escape, the Vscode-Neovim extension doesn't interfere with anything I'm doing (as it would if I were changing the buffer without this feature). 
The feature actually works fine, but let's see.... have not checked the problems in #642